configuration/README.md

```diff
@@ -28,19 +28,21 @@ Sources are identified via regexp according to package, type, and field names.
 }
 ```
 
-Sinks and sanitizers are identified via regexp according to package and method name.
+Sinks, sanitizers, and some propagators (below) are identified via regexp according to package, method, and (optional) receiver name.
 
 ```json
 {
   "Sinks": [
     {
       "PackageRE": "<package path regexp>",
+      "ReceiverRE": "<type name regexp>",
       "MethodRE": "<method name regexp>"
```

Comment:
I think it would make more sense to call this field "FunctionRE" or something similar. Indeed if there is an explicit receiver, then it is a method, but if there isn't, then it is a function, but methods are functions, so "FunctionRE" would capture both.

Reply:
That's a good point.  That's kind of beyond the scope of this PR though.  I'll open a rename PR after this one.